content/_partners/our-services._en.md

```diff
@@ -8,42 +8,52 @@ title: >-
+enhanced_identity_verification: >-
+    ### Enhanced identity verification (IAL2 compliant)
 
-    Login.gov provides customer support through our contact center services in English, Spanish, and French (through a translation service) 24 hours a day, seven days a week, excluding federal holidays. Most inquiries are received from the Login.gov contact form and answered by email within 2 business days.
 
+    For this service level, users create an authentication-only account, and then go further by supplying identity documentation and a selfie. Login.gov uses proven facial matching technology that compares the selfie exclusively with the user’s photo ID—and does not use the image for any other purpose. Users also have the option to choose in-person identity verification at a participating U.S. Postal Service location. The enhanced identity verification service has been certified as compliant at the [NIST IAL2 level](https://pages.nist.gov/800-63-3/){:class="usa-link--external"} by an independent, third-party assessor [Kantara Initiative](https://kantarainitiative.org/trust-status-list/){:class="usa-link--external"}.
 
-    For more complex issues, our agents are available to answer telephone calls as well. Any needed services outside these hours are addressed on a case by case basis depending on partner needs.
 
+    ![Kantara logo](/assets/img/partners/kantara_logo.png){: .display-block .margin-x-auto}
+end_user_support: >-
+    ### End user support
 
-    We also provide information to partner agency help desks and support teams to help end-users who may contact the agency with questions.
+
+    Login.gov provides support to end users by offering a comprehensive [online help center](https://www.login.gov/help/){:class="usa-link--external"} with articles and FAQs addressing common issues - available in English, Spanish, French, and Simplified Chinese. We also operate a help desk 24 hours a day, seven days a week with agents that answer inquiries from our [contact form](https://www.login.gov/contact/){:class="usa-link--external"} and telephone calls for more complex issues, including in multiple languages via interpreters. Our team will also work together with your agency help desk to assist in providing help content for your end users. 
+accessibility_multilingual_support: >-
+    ### A focus on accessibility and multilingual support
+
+
+    Login.gov prioritizes [accessibility](https://www.login.gov/accessibility/), including an emphasis on Section 508 compliance, to support users with disabilities. Our application is designed to be compatible with screen readers, keyboard navigation, and other assistive technologies that make it easier for individuals with visual, motor, or cognitive impairments to access services.
```

Comment:
I'd suggest using relative paths so these URLs link to the same domain in preview sites and local development, or optionally incorporting `{{ site.baseurl }}` if you wanted fully-qualified URLs.

```suggestion
    Login.gov provides support to end users by offering a comprehensive [online help center](/help/){:class="usa-link--external"} with articles and FAQs addressing common issues - available in English, Spanish, French, and Simplified Chinese. We also operate a help desk 24 hours a day, seven days a week with agents that answer inquiries from our [contact form](/contact/){:class="usa-link--external"} and telephone calls for more complex issues, including in multiple languages via interpreters. Our team will also work together with your agency help desk to assist in providing help content for your end users. 
accessibility_multilingual_support: >-
    ### A focus on accessibility and multilingual support


    Login.gov prioritizes [accessibility](/accessibility/), including an emphasis on Section 508 compliance, to support users with disabilities. Our application is designed to be compatible with screen readers, keyboard navigation, and other assistive technologies that make it easier for individuals with visual, motor, or cognitive impairments to access services.
```